core/src/main/java/com/rim/IceField/PlayerBase.java

```diff
@@ -26,31 +26,36 @@ public void SavePlayer(PlayerBase player)
     {
 		System.out.println("Player has been saved!");
     }
-
-    //Had to make this method boolean in order to be able to check if the Eskimo constructed the igloo
-    public boolean useSkill()
+  
+    public void useSkill()
     {
-        System.out.println("Empty Skill.");
-    }
+       System.out.println("Empty Skill.");
+		}
+
 
     public void useItem()
     {
```

Comment:
I guess here you get passed already an argument, say ItemBase item, if you have it in the item base (search by id in inventory) you call use item from itemBase. The input from the user would be passed to you, so no need to worry about it :)